js/src/apbct-public.js

```diff
@@ -268,11 +268,74 @@
 			}
 
 		}, 1000);
+
+		// Listen clicks on encoded emails
+		let decodedEmailNodes = document.querySelectorAll("[data-original-string]");
+		if (decodedEmailNodes.length) {
+			for (let i = 0; i < decodedEmailNodes.length; ++i) {
+				if (
+					decodedEmailNodes[i].parentElement.href ||
+					decodedEmailNodes[i].parentElement.parentElement.href
+				) {
+					// Skip listening click on hyperlinks
+					continue;
+				}
+				decodedEmailNodes[i].addEventListener('click', function ctFillDecodedEmailHandler(event) {
+					this.removeEventListener('click', ctFillDecodedEmailHandler);
+					apbctAjaxEmailDecode(event);
+				});
+			}
+		}
 	}
 	apbct_attach_event_handler(window, "DOMContentLoaded", apbct_ready);
 
 }());
 
+function apbctAjaxEmailDecode(event){
+	const element = event.target;
+	element.setAttribute('title', 'Wait for decoding...');
```

Comment:
the text will be without translation

Reply:
Done https://github.com/CleanTalk/wordpress-antispam/commit/79013d756cb0ee12e42616f84c0655c98f441e08